### PR TITLE
content update 

### PR DESCRIPTION
**Pull request by DannyS03**
Addresses: https://github.com/OpenLiberty/guide-rest-client-angularjs/issues/16

### CODE REVIEW

**Comment by DannyS03** [comment]
> @DannyS03
> 
> **Creating the AngularJS template**
> 
> * [ ]  add hotspots to `angular.js` and `angular-resource.js`

The reference should say "script" rather than "file", as AngularJS is being called into the HTML page and there are not actual `angular.js` or `angular-resource.js` files in the application. I updated `file` to `script`, @gkwan-ibm.

**Comment by gkwan-ibm** [comment]
@DannyS03 you say "... defining the `angular.js` script in your HTML..." where is it? Should we highlight it? Same as the "...defining the `angular-resource.js` script in your HTML..."

**Comment by DannyS03** [comment]
> @DannyS03 you say "... defining the `angular.js` script in your HTML..." where is it? Should we highlight it? Same as the "...defining the `angular-resource.js` script in your HTML..."

Yes, we should highlight it. I created two new hotspots in the index.html.

**Comment by gkwan-ibm** [comment]
hi @Charlotte-Holt would you review the README changes?

File: README.adoc [hunk 1]
```diff
@@ -90,10 +90,10 @@ Navigate to the `start` directory to begin.
 [role='command']
 include::{common-includes}/devmode-start.adoc[]
 
-After you start the server, you can find your artist JSON at the following URL: http://localhost:9080/artists[http://localhost:9080/artists^].
+Once the server is started, you can find your artist JSON at the following URL: http://localhost:9080/artists[http://localhost:9080/artists^].
```
Comment (Charlotte-Holt) [inline README.adoc:93]: Update

>Once the server is started, you can find your artist JSON at the following URL: http://localhost:9080/artists[http://localhost:9080/artists^].

To

>After the server is started, you can find your artist JSON at the following URL: http://localhost:9080/artists[http://localhost:9080/artists^].

File: README.adoc [hunk 2]
```diff
@@ -199,9 +199,7 @@ was consumed correctly.
 
 When you are done checking the application root, exit development mode by pressing CTRL+C in the command-line session where you ran the server, or by typing q and then pressing the enter/return key.
 
-Although this guide did not teach you how to build logic, you will likely build logic when you develop your own
-applications, and testing will become a crucial part of your development lifecycle. If you need
-to write test cases, follow the official unit testing and end-to-end testing documentation on the
+When you develop your own applications, and testing will become a crucial part of your development lifecycle. If you need to write test cases, follow the official unit testing and end-to-end testing documentation on the
```
Comment (Charlotte-Holt) [inline README.adoc:202]: Update

>When you develop your own applications, and testing will become a crucial part of your development lifecycle. If you need to write test cases, follow the official unit testing and end-to-end testing documentation on the official AngularJS page.

To

>When you develop your own applications, testing becomes a crucial part of your development lifecycle. If you need to write test cases, follow the official unit testing and end-to-end testing documentation on the official AngularJS website.

File: README.adoc [hunk 3]
```diff
@@ -164,8 +164,8 @@ include::finish/src/main/webapp/js/consume-rest.js[]
 
 Before your application is bootstrapped, you must pull in two [hotspot=AngularJS file=0]`AngularJS` libraries and import [hotspot=consume-rest file=0]`consume-rest.js`.
 
-The first import is the base AngularJS library. The second library is responsible for providing the APIs
-for the `$resource` service. The application is bootstrapped because the [hotspot=consumeRestApp file=0]`consumeRestApp` application module is attached to the [hotspot=body file=0]`body` of the template.
+The first import is the base AngularJS library, defining the [hotspot=angular-script file=0]`angular.js` script in your HTML. The second import is the library responsible for providing the APIs for the `$resource` service, defining the 
+[hotspot=angular-resource-script file=0]`angular-resource.js` script in your HTML. The application is bootstrapped because the [hotspot=consumeRestApp file=0]`consumeRestApp` application module is attached to the [hotspot=body file=0]`body` of the template.
```
Comment (Charlotte-Holt) [inline README.adoc:168]: Update

>The first import is the base AngularJS library, defining the `angular.js` script in your HTML. The second import is the library responsible for providing the APIs for the `$resource` service, defining the `angular-resource.js` script in your HTML.

To

>The first import is the base AngularJS library, which defines the `angular.js` script in your HTML. The second import is the library that's responsible for providing the APIs for the `$resource` service, which also defines the `angular-resource.js` script in your HTML.

File: README.adoc (hunk 1 above)
Comment (DannyS03) [reply README.adoc]: Addressed.

File: README.adoc (hunk 2 above)
Comment (DannyS03) [reply README.adoc]: Addressed.

File: README.adoc (hunk 3 above)
Comment (DannyS03) [reply README.adoc:168]: Addressed.

**Comment by AustinSeto** [comment]
Checked on QA site. Everything looks good except for a small issue in section "What you'll learn" where the following message appears:

```
Unresolved directive in README.adoc - include::https://raw.githubusercontent.com/OpenLiberty/guides-common/master/gitclone.adoc[]
```

This part of the guide was not affected by the changes in this PR so I think this is an issue with the QA site and not this change.

**Pull request merged**